restler/engine/core/request_utilities.py

```diff
@@ -74,8 +74,9 @@ def execute_token_refresh_cmd(cmd):
             else:
                 cmd_result = subprocess.getoutput([cmd])
 
-            _, latest_token_value, latest_shadow_token_value = parse_authentication_tokens(cmd_result)
             _RAW_LOGGING(f"New value: {cmd_result}")
+            _, latest_token_value, latest_shadow_token_value = parse_authentication_tokens(cmd_result)
+            _RAW_LOGGING(f"Parse latest token value succeed: {cmd_result}")
```

Comment:
>succeed: {cmd_result} [](http://example.com/codeflow?start=52&length=21)

Since it's printing the same output as above, could you please remove this part, and just print something like 'Successfully obtained the latest token'?

Reply:
Good. Done.